scripting/get5_apistats.sp

```diff
@@ -184,6 +199,7 @@ static void CheckForLogo(const char[] logo) {
                      : CreateRequest(k_EHTTPMethodGET, "/static/img/logos/%s.png", logo);
 
     if (req == INVALID_HANDLE) {
+      LogDebug("Failed start the logo API request");
```

Comment:
Redundant LogDebug as the built-in `http.sp` method already logs if it returns an invalid handle.

Reply:
done